### PR TITLE
Hw1 mvc

### PR DESCRIPTION
**Pull request by TetyanaKarpikova**

### CODE REVIEW

File: dashboard.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```
Comment (matthewlaz) [inline dashboard.html:1]: can't get what this file for

Comment (matthewlaz) [reply dashboard.html:1]: Also index.js in project

File: signIn.html [hunk 2]
```diff
@@ -18,29 +18,30 @@
...
                 </button>
             </div>
             <div class="wrapper-send-form">
                 <button class="wrapper-send-form__button" type="submit">Sign In ></button>
             </div>
         </form>
         <div class="wrapper-link">
-            <span> Don't have an account?</span>            
+            <span> Don't have an account?</span>
             <a href="./signUp.html">Sign up</a>
```
Comment (matthewlaz) [inline signIn.html:41]: Idk your task, but if it is history api, this is not about it

File: mvc.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        this.data = JSON.parse(localStorage.getItem("signUp")) || [];
+    }
+
+    setData(user) {
+        localStorage.setItem("signUp", JSON.stringify(user));
+    }
+
+    addData(email, password) {
+        const newUser = {
+            email,
```
Comment (matthewlaz) [inline mvc.js:12]: in Sign Up form you have 5 fields. I think you should push 4 fields to storage(first and last name, email  and password)Confirm is not necessary

File: dashboard.html (hunk 1 above)
Comment (TetyanaKarpikova) [reply dashboard.html:1]: dashboard.html - this file opens when the user correctly writes email and password on the signin page (empty file).
index.html - this file was need, maybe, for write routers code.

File: signIn.html (hunk 2 above)
Comment (TetyanaKarpikova) [reply signIn.html:41]: history api - I`ll write in the next step.

File: dashboard.html (hunk 1 above)
Comment (matthewlaz) [reply dashboard.html:1]: Ok

File: signIn.html (hunk 2 above)
Comment (matthewlaz) [reply signIn.html:41]: Resolved

File: mvc.js (hunk 3 above)
Comment (matthewlaz) [reply mvc.js:12]: Resolved

**Pull request merged**